Makefile

```
@@ -75,22 +75,14 @@ clean:
 
 binaries: critest crictl
 
-install-critest:
-	install -D -m 755 $(CURDIR)/_output/critest$(BIN_EXT) $(BINDIR)/critest$(BIN_EXT)
+install-critest: critest
```

Can you try to drop the dependency:
```suggestion
install-critest:
```

---

I don't have better ideas either, let's drop the dependency of critest/crictl

---

fixed by #689